src/emoji.ts

```diff
@@ -0,0 +1,22 @@
+import { CIResult } from "./pr-info";
+
+export function failure(passed: boolean) {
+    return passed ? "✅" : "❌";
+}
+
+export function pending(passed: boolean) {
+    return passed ? "✅" : "🕐";
+}
+
+export function result(result: CIResult) {
+    switch (result) {
+        case "fail":
+            return { emoji: "❌", text: "have failed" };
+        case "pass":
+            return { emoji: "✅", text: "have passed" };
+        case "unknown":
+            return { emoji: "🕐", text: "are still running" };
+        default:
```

Comment:
Should be `case "missing"` so there's a type error if the type gets extended in the future.